Source/ACE.Server/WorldObjects/WorldObject_Database.cs

```diff
@@ -132,6 +155,9 @@ public bool IsStaticThatShouldPersistToShard()
         /// <returns></returns>
         public bool IsDynamicThatShouldPersistToShard()
         {
+            if (DoNotSave)
```

Comment:
DoNotSave is not a condition for this function

Reply:
What do you mean is not a condition for this function? It checks the WorldObject property DoNotSave which is set in the god function. Not being hostile - legitimately trying to learn. Are these static (and the dynamic one below) not related to players, only for items placed on hooks or in chests?

Reply:
This function returns a bool that indicates if the wo is the type of object that should persist to shard.

It does not return the bool that indicates both should persist to shard and can persist to shard.

Reply:
Understood. This check is not needed here. Thank you.